clickhouse/client.h

```diff
@@ -69,6 +69,13 @@ struct ClientOptions {
     DECLARE_FIELD(tcp_keepalive_intvl, std::chrono::seconds, SetTcpKeepAliveInterval, std::chrono::seconds(5));
     DECLARE_FIELD(tcp_keepalive_cnt, unsigned int, SetTcpKeepAliveCount, 3);
 
+    /// Socket timeout
+    DECLARE_FIELD(connection_socket_timeout, bool, ConnectionTimeout, false);
+    DECLARE_FIELD(connection_socket_recv_timeout_sec, unsigned int, SetConnectionRecvTimeoutSeconds, 60);
```

Comment:
1. I think seconds precision timeout is enough.
2. Please use std::chrono::seconds as in all other cases.

Reply:
1. remove usec precision
2. change type of timeouts to std::chrono::seconds